node_modules.diff

```diff
@@ -0,0 +1,144 @@
+diff --git a/test/dynamic-plugin-sdk-utils/dist/index.js b/test/dynamic-plugin-sdk-utils/dist/index.js
```

Comment:
The `node_modules` diff at the end of the work in the [debugging commit](https://github.com/openshift/hac-core/pull/43/commits/8812ae218413a3d847ef53ea5e59e54775b79485).